pkg/services/auth/internal/gitlab.go

```diff
@@ -38,6 +38,10 @@ func getEnvDefault(key, defaultValue string) string {
 	return val
 }
 
+func getGitlabHost() string {
+	return getEnvDefault("GITLAB_HOSTNAME", gitlabHost)
+}
```

Comment:
Allow configuring where to send oauth requests..

Reply:
Should we be documenting this somewhere for users?

Reply:
Not yet it doesn't feel like a very complete feature. See PR description, it becomes the global HOST for all gitlab oauth requests. I could make some follow up issues though.